Fix typo (#6207)
segmentio/segment-docs: src/connections/destinations/catalog/google-tag-manager/index.md

For Google Tag Manager, consent mode settings need to be managed directly [within your GTM account](https://support.google.com/tagmanager/answer/10718549?hl=en#tag-settings){:target="_blank"}. There's no direct update from Segment for the GTM destination regarding consent mode, as it's managed within GTM tags themselves.

> Segment recommends you to install a consent management platform that uses the current [consent-tools wrapper](https://github.com/segmentio/analytics-next/tree/master/packages/consent/consent-tools){:target="_blank"} that's outside of Google Tag Manager such as [One Trust](https://tanelytics.com/integrate-onetrust-with-google-tag-manager/){:target="_blank"}.

Reviewer: ```suggestion
Segment recommends you install a consent management platform that uses the current [consent-tools wrapper](https://github.com/segmentio/analytics-next/tree/master/packages/consent/consent-tools){:target="_blank"} that's outside of Google Tag Manager like [OneTrust](https://tanelytics.com/integrate-onetrust-with-google-tag-manager/){:target="_blank"}.
```
